Makefile

```diff
@@ -41,3 +37,8 @@ schools_raw.csv:
 		TO STDOUT WITH CSV HEADER' > build/$@
 		touch build/$@
 
+# Create the main csv file with human readble data from IPEDS
+process:
```

Comment:
The target should be `build/schools_processed.csv` and it should have a dependency `build/schools_processed.csv`